packages/blue-sdk-viem/src/fetch/Market.ts

```diff
@@ -62,38 +63,18 @@ export async function fetchMarket(
     }
   }
 
-  const [loanToken, collateralToken, oracle, irm, lltv] = await readContract(
-    client,
-    {
+  const params = new MarketParams(
```

Comment:
It makes me think we could improve speed of such queries for idle markets because `params` is only used afterwards to fetch the oracle price
Something for another PR I guess